src/stars.css

```
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+  width: 96px;
+}
+
+.stars__star {
+  background-image: url(https://raw.githubusercontent.com/NadieinOleh/layout_product-cards/059be13cae6c157e2b2c9b9f75352d92be91c02e/src/images/star-active.svg);
```

```suggestion
  background-image: url(images/star-active.svg);
```
you can use relative path